pbs/pbsrequest_test.go

```diff
@@ -109,6 +112,9 @@ func TestParseSimpleRequest(t *testing.T) {
 	if pbs_req.AdUnits[1].MediaTypes[1] != "video" {
 		t.Errorf("Instead of video MediaType received %s", pbs_req.AdUnits[1].MediaTypes[0])
 	}
+	if pbs_req.AdUnits[1].Video.Mimes[0] != "video/mp4" {
```

Comment:
Maybe we should make a list of supported mime types instead of hardcoding "video/mp4" here.

Reply:
ORTB doesn't give such a list, so I imagine the list would be a bit extensive. But if you have such a list I'd add it here :)

Reply:
It doesn't have to be an _exhaustive_ list--especially not if your changes only use this one... but it's probably a good idea to at least make this a `const`.

If we never use another, then you've at least avoided a magic string. If we find ourselves adding more in the future, they're extracted from the code and easier to consolidate.